docker-compose.template

```diff
@@ -14,7 +14,7 @@ solr:
     # hydroshare repository
     - "HS_PATH:/hydroshare"
   ports:
-    - "8983"
+    - "8983:8983"
```

Comment:
@xuezhaokun It is OK to open the solr port up in your local dev env for debugging purposes, but do not check in the change since this port is not supposed to open in production environment.